toolchain/macosx.cmake

```diff
@@ -0,0 +1,36 @@
+set(CMAKE_INSTALL_PREFIX /opt/sysroot)
+set(CMAKE_SYSROOT /opt/sysroot)
+set(CMAKE_SYSTEM_NAME Darwin)
+set(CMAKE_SYSTEM_PROCESSOR x86_64)
+
+set(triple x86_64-apple-darwin)
+
+set(CMAKE_C_COMPILER clang)
+set(CMAKE_C_COMPILER_TARGET ${triple})
+set(CMAKE_CXX_COMPILER clang++)
+set(CMAKE_CXX_COMPILER_TARGET ${triple})
+
+set(CMAKE_EXE_LINKER_FLAGS "-mtvos-version-min=9.2")
```

Comment:
This could be an issue. I'm not sure what might be needed for osx. Maybe take a look at osxcross